### PR TITLE
verify correctness of dy2static model by using paddle-inference api

### PR DESCRIPTION
**Pull request by OliverLPH**
### PR types

Others

### PR changes

Others

### Describe

test correctness of dy2static model when using paddle-inference api.
make sure train model -> save model -> deploy model process is correct.

Include following models
BERT
BMN
LAC
MobileNet V1/V2
ResNet
SeResNet
Mnist

### CODE REVIEW

**Review by vslyu — DISMISSED** [review]

File: python/paddle/fluid/tests/unittests/dygraph_to_static/predictor_utils.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+import numpy as np
+import paddle
+import paddle.fluid as fluid
+
+from paddle.fluid.core import AnalysisConfig
+from paddle.fluid.core import create_paddle_predictor
+
+
+class PredictorTools(object):
+    ''' Paddle-Inference predictor
```
Comment (zhhsplendid) [inline python/paddle/fluid/tests/unittests/dygraph_to_static/predictor_utils.py]: Keep same doc style in one project

Let's write:
```
'''
Paddle-Inference predictor
'''
```

File: python/paddle/fluid/tests/unittests/dygraph_to_static/predictor_utils.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    '''
+
+    def __init__(self, model_path, params_file, feeds_var):
+        '''__init__
```
Comment (zhhsplendid) [inline python/paddle/fluid/tests/unittests/dygraph_to_static/predictor_utils.py]: Keep same doc style in one project

Let's write:
```
'''
__init__
'''
```

File: python/paddle/fluid/tests/unittests/dygraph_to_static/predictor_utils.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        output_names = predictor.get_output_names()
+        outs = [
+            predictor.get_output_tensor(out_name).copy_to_cpu()
+            for out_name in output_names
+        ]
+
+        return outs
+
+    def __call__(self):
+        '''__call__
```
Comment (zhhsplendid) [inline python/paddle/fluid/tests/unittests/dygraph_to_static/predictor_utils.py]: Keep same doc style in one project

Let's write:
```
'''
__call__
'''
```

**Review by zhhsplendid — DISMISSED** [review]
Overall LGTM, only some comment issues. You can choose to change in this PR or next PR.

Approved.

**Comment by OliverLPH** [comment]
@zhhsplendid 
thanks for your advice, I will fix doc style in this PR

File: python/paddle/fluid/tests/unittests/dygraph_to_static/predictor_utils.py (hunk 1 above)
Comment (OliverLPH) [reply python/paddle/fluid/tests/unittests/dygraph_to_static/predictor_utils.py]: done

File: python/paddle/fluid/tests/unittests/dygraph_to_static/predictor_utils.py (hunk 2 above)
Comment (OliverLPH) [reply python/paddle/fluid/tests/unittests/dygraph_to_static/predictor_utils.py]: done

File: python/paddle/fluid/tests/unittests/dygraph_to_static/predictor_utils.py (hunk 3 above)
Comment (OliverLPH) [reply python/paddle/fluid/tests/unittests/dygraph_to_static/predictor_utils.py]: done

**Review by zhhsplendid — APPROVED** [review]
LGTM

**Pull request merged**